.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+
+node_js:
+  - "stable"
+  
+cache:
+  directories:
+    - node_modules
+    
+install:
+  - yarn install
+
+script:
+  - yarn test -- --coverage
```

Comment:
Hey, based on @Arthur236's feedback. Change this to `yarn coverage`

Reply:
Okay, that's noted